inginious/frontend/app.py

```diff
@@ -283,7 +289,7 @@ def flask_internalerror(e):
     flask_app.register_error_handler(InternalServerError, flask_internalerror)
 
     # Insert the needed singletons into the application, to allow pages to call them
-    flask_app.get_homepath = get_homepath
+    flask_app.get_path = get_path
```

Comment:
`get_path` should now be used in all instances where `get_homepath` was used to build an URL. Templates still have access to `get_homepath` when a public URL is needed, such that I didn't have to go thru all that code (esp. layout.html) by hand.